packages/language-server/src/completion-items.ts

```diff
@@ -252,6 +257,43 @@ function createTextEditsForClassInTagName(
   return { range, newText, filterText };
 }
 
+function addClosingTagTextEdit(
+  xmlElement: XMLElement,
+  closingTagName: string,
+  additionalTextEdits: TextEdit[]
+): void {
+  // Note: if we implement syncronization between the opening and closing tag in the generic XML extension
+  // and it will cover changes done by code assist suggestions this logic should be removed from this extension
+  if (xmlElement.syntax.closeName !== undefined) {
+    // Replace name in closing tag.
+    // Note: in case of invalid xml where the current tag is closed but has no closing tag,
```

Comment:
so you need to know if its self closing?

Reply:
https://github.com/SAP/xml-tools/blob/master/packages/ast/CHANGELOG.md#410-2020-05-05

Reply:
No, I mean this use case:
```xml
<parentTag>
  <innerTag>
</parentTag>
```

`</parentTag>` is considered the closing tag of `<innerTag>`